chart/templates/_helpers.yaml

```
@@ -204,7 +208,11 @@ If release name contains chart name it will be used as a full name.
     mountPath: /git
   {{- if .Values.dags.gitSync.sshKeySecret }}
   - name: git-sync-ssh-key
+    {{- if empty .Values.dags.gitSync.sshKeyCustomFile }}
     mountPath: /etc/git-secret/ssh
+    {{- else }}
+    mountPath: {{.Values.dags.gitSync.sshKeyCustomFile | quote}}
+    {{- end }}
```

```suggestion
    mountPath: {{ .Values.dags.gitSync.sshKeyCustomFile | default "/etc/git-secret/ssh" | quote }}
```

I think this is better?